src/util/customTransformer.js

```diff
@@ -36,24 +36,29 @@ async function runUserTransform(
   await jail.set(
     '_fetch',
     new ivm.Reference(async (resolve, ...args) => {
+      const fetchStartTime = new Date();
+      const fetchTags = { ...trTags };
       try {
-        const fetchStartTime = new Date();
         const res = await fetchWithDnsWrapper(trTags, ...args);
         const data = await res.json();
-        stats.timing('fetch_call_duration', fetchStartTime, trTags);
+        fetchTags.isSuccess = true;
```

Comment:
Are booleans supported as label values? [This](https://github.com/siimon/prom-client/blob/master/index.d.ts#L221-L223) suggests that only strings and numbers are supported